src/app/content/studyGuides/components/Filters.tsx

```diff
@@ -31,13 +32,16 @@ const ConnectedFilterList = connect(
   })
 )(FiltersList);
 
-export default () =>
-  <Filters>
+export default () => {
+  const userLoggedOut = useSelector(loggedOut);
```

Comment:
this is the correct place for the auth logic, from here just pass `disabled` props down where they are needed